samples/attested_tls/README.md

```diff
@@ -72,6 +73,8 @@ Note that there are two different build systems supported, one using GNU Make an
 
 You can go build from the sample's root directory or go to tls_between_enclaves or tls_between_non_enclave_enclave subdirectory to build each sub-sample
 
+Note: This sample uses an OE SDK customized version of mbedtls library for TLS channel connection. It has MBEDTLS_NET_C component enabled, which has a dependecy on the newly added [socket support](../../docs/UsingTheIOSubsystem.md#socketh) in 0.6.0 OE SDK release (for more details see [Using the Open Enclave I/O subsystem](../../docs/UsingTheIOSubsystem.md#opting-in) for details). So in order to build successfully, you would need to link with liboehostsock and libhostresolver libraries to satisfy the dependency.
```

Comment:
Too late feedback - but dependency is spelt incorrectly.
Thanks for adding the details on linking the libraries which was missing earlier.